tpp-prophets/protein_prophet.xml

```diff
@@ -1,21 +1,30 @@
 <tool id="proteomics_search_protein_prophet_1" name="Protein Prophet" version="1.1.1">
     <description>Calculate Protein Prophet statistics on search results</description>
     <requirements>
-        <container type="docker">iracooke/protk-1.4.3</container>
         <requirement type="package" version="1.4.4a">protk</requirement>
     </requirements>
-    <command>
+    <command detect_errors="aggressive">
+    <![CDATA[
+        ln -s '$database' ./AASequences.fasta &&
+        #for $counter, $input_file in enumerate($input_files):
+            ln -s '$input_file' ${counter}.xml &&
+            sed -i.bak 's|<search_database local_path=".*" type="AA"\/>|<search_database local_path="\./AASequences\.fasta" type="AA"\/>|g' ${counter}.xml &&
```

Comment:
This is my ugly hack to change the XML on the fly and reference the use-given database.

Reply:
Hmm ... this is very clever but I am worried that it will break the tool under normal use.  This should only be needed for tests.  In normal use the database paths will all be set correctly.  My preference would be to have the hack only get run for the test case